test/pipeline_test.dart

```diff
@@ -8,76 +8,53 @@ import 'package:test/test.dart';
 import 'test_util.dart';
 
 void main() {
-  test('compose middleware with Pipeline', () async {
-    var accessLocation = 0;
-
-    var middlewareA = createMiddleware(requestHandler: (request) {
-      expect(accessLocation, 0);
-      accessLocation = 1;
-      return null;
-    }, responseHandler: (response) {
-      expect(accessLocation, 4);
-      accessLocation = 5;
-      return response;
-    });
-
-    var middlewareB = createMiddleware(requestHandler: (request) {
-      expect(accessLocation, 1);
-      accessLocation = 2;
-      return null;
-    }, responseHandler: (response) {
-      expect(accessLocation, 3);
-      accessLocation = 4;
-      return response;
-    });
+  var accessLocation = 0;
```

Comment:
nit: initialize this to zero in a `setUp` method instead of at the top of each test

Reply:
done